app/Models/Trade.php

```diff
@@ -34,6 +36,10 @@ class Trade extends Model
         'character_id' => 'integer',
     ];
 
+    public const STATUS = ["open","closed"];
```

Comment:
this should probably be 2 constants for less ambiguity 
```suggestion
    public const STATUS_OPEN = "open";
    public const STATUS_CLOSED = "closed";
```